specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1505 @@
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
+            }
+          },
+          "201": {
+            "description": "NetworkVerifier was created",
+            "schema": {
+              "$ref": "#/definitions/NetworkVerifier"
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkVerifierPut": {
+            "$ref": "./examples/NetworkVerifierPut.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetworkVerifier"
+        ],
+        "summary": "Deletes the specific Network Verifier resource asynchronously",
+        "operationId": "NetworkVerifierDelete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "networkVerifierName",
+            "description": "The name of the Network Verifier resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "NetworkVerifier delete request accepted and will complete asynchronously"
+          },
+          "204": {
+            "description": "NetworkVerifier delete request succeeded"
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "x-ms-examples": {
+          "NetworkVerifierDelete": {
+            "$ref": "./examples/NetworkVerifierDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnalysisIntent": {
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/AnalysisIntentProperties"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "location": {
+          "type": "string"
```

Comment:
use the definitions from common types : [https://github.com/Azure/azure-rest-api-specs/blob/release-network-2022-06-01-preview/specification/common-types/resource-management/v3/types.json](https://github.com/Azure/azure-rest-api-specs/blob/release-network-2022-06-01-preview/specification/common-types/resource-management/v3/types.json)

Reply:
Used the TrackedResource common type.